src/includes/performance/enable-tracing/android.mdx

```diff
@@ -0,0 +1,38 @@
+```xml {filename:AndroidManifest.xml}
+<application>
+  <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+</application>
+```
+
+Or, if you are manually instrumenting Sentry:
+
+```java {tabTitle:Java}
```

Comment:
What about having Kotlin first?

Maybe better as a single PR across all of docs, right?

Reply:
I'd say so, also would need checking if docs somehow would order alphabetically or not